src/core_modules/capture-core/components/LockedSelector/QuickSelector/ActionButtons.component.js

```diff
@@ -16,89 +17,86 @@ const styles = () => ({
         flexGrow: 1,
     },
     icon: {
-        paddingRight: 5,
+        fontSize: 20,
+    },
+    buttonMargin: {
+        marginLeft: 8,
+    },
+    rightButton: {
+        marginLeft: 12,
     },
 });
 
-type Props = {
-    classes: Object,
-    selectedProgram: string,
-    onStartAgain: () => void,
-    onClickNew: () => void,
+type Props = $ReadOnly<{|
+    selectedProgramId: string,
+    dispatchStartAgainClick: () => void,
+    dispatchNewClick: () => void,
+    dispatchFindClick: () => void,
```

Comment:
You are suggesting the naming to be like this. 

Same for me but do you feel that it reads nice now? "On click = on click start again" vs "On click = dispatch start again"
![image](https://user-images.githubusercontent.com/4181674/92915966-be37e680-f435-11ea-8484-0cfa018b2bc5.png)

Reply:
For the component (`ActionButtons.component` ) it shouldn't matter if we're using Redux or not. If we name the prop `dispatchStartAgainClick` it implies that we are using Redux. `onStartAgainClick` is more generic and could be used whether we use Redux, pure React state management or whatever solution we like.

Reply:
I am applying the change. This was only a suggestion to begin with. 

However dispatch is simply a word. It could imply redux and it also it couldnt. What it implies is that this action is sent to another place. We could use Mobx and have functions named dispatch, we could use plan HTML DOM Events and use the word dispatch. 

Another though I have is that perhaps discussing the replacement of redux with another state management library is a bit of a stretch because doing that implies already _many_ changes in the code in the first place.